sympy/printing/str.py

```diff
@@ -74,13 +74,16 @@ def _print_BooleanTrue(self, expr):
     def _print_BooleanFalse(self, expr):
         return "False"
 
+    def _print_Not(self, expr):
+        return '~(%s)' %(expr.args[0])
+
     def _print_And(self, expr):
-        return '%s(%s)' % (expr.func, ', '.join(sorted(self._print(a) for a in
-            expr.args)))
+        return '%s' %(' & '.join([self.parenthesize(arg, 38)
+            for arg in expr.args]))
 
     def _print_Or(self, expr):
-        return '%s(%s)' % (expr.func, ', '.join(sorted(self._print(a) for a in
-            expr.args)))
+        return '%s' %(' | '.join([self.parenthesize(arg, 36)
+            for arg in expr.args]))
```

Comment:
Shall I put in a comment like this : 
```python
# The number 36 has been used (instead of PRECEDENCE["Or"]) as 
# Bitwise or operator (|) is used to represent Or objects in SymPy
# and not Boolean or (or).
```